r/R/rzeppelin/R/common.R

```diff
@@ -0,0 +1,14 @@
+strintrplt <- function(snippet,envir=parent.frame()) {
```

Comment:
These are all the same thing --- rscala originally has two-way communication.  We don't need that, we only need one-way.  So I took out the functions that provided communication in the opposite direction.  That's the difference you're seeing.  (There are also some purely mechanical changes, like to the name of the package.)  Since this is only deletions and mechanical changes, not the addition or modification of anything that isn't de minimus, its all still under rscala's BSD license.  

Think about it this way:  Imagine that a single BSD-licensed source file had two functions in it.  I delete one, and replace it with a completely re-written function.  So now we have a file that is half BSD-licensed, and half ASF-licensed.  Right?  Now imagine that instead of re-writing one of the two functions, I just delete one.  We've changed the file, and it looks different, but its still completely BSD-licensed, because we haven't done anything that the other license could apply to.  

Does this make sense?